tests/components/broadlink/test_device.py

```diff
@@ -49,7 +50,7 @@ async def test_device_setup_authentication_error(hass):
     ) as mock_forward, patch.object(
         hass.config_entries.flow, "async_init"
     ) as mock_init:
-        mock_api, mock_entry = await device.setup_entry(hass, mock_api=mock_api)
+        mock_api, mock_entry, _ = await device.setup_entry(hass, mock_api=mock_api)
```

Comment:
`mock_api, mock_entry, mock_discovery` are asking for their own namespace. We can use a namedtuple in a future clean up.

Reply:
@MartinHjelmare Can I do it here or should I open another PR?